torch/_inductor/utils.py

```diff
@@ -586,6 +586,11 @@ def has_incompatible_cudagraph_ops(gm):
         "run_and_save_rng_state",
         "run_with_rng_state",
         "aten._local_scalar_dense",
+        # Technically, it's not necessary to ban this, because an
+        # assert_scalar with constant arguments can be validly run
+        # with CUDA graphs, but the operator is also pointless with
+        # constant arguments, so might as well ban
+        "aten._assert_scalar",
```

Comment:
Can we verify that if this has constant arguments it won't get generated ?

Reply:
I'm not sure concretely what you are suggesting? Like, nothing in FX is stopping you from inserting an assert_scalar with a constant argument. So this is more of an invariant for whatever is generating the graphs. But that's mostly... did I mess up the `assert_scalar` insertion. Which I don't think I did? lol